lib/simple_config.py

```diff
@@ -288,7 +288,7 @@ def has_fee_estimates(self):
         return len(self.fee_estimates)==4
 
     def fee_per_kb(self):
-        return self.get('fee_per_kb', self.max_fee_rate()/2)
+        return self.get('fee_per_kb', self.fee_rates[0])
```

Comment:
I think we're better just putting 1000 here no?  The array may change and affect this undesirably.

Reply:
If you disagree strongly with this I can hard-code it.  

I was just afraid if it was a hard-coded value that later ended up falling "out of range" of the fee slider it would produce weird UI behavior.

Reply:
Add a decent comment and the concerns about thoughtless changes become unwarranted.

Reply:
Also, max(fee_slider.minimum, 1000) prevents out of range from happening.

Reply:
Gotcha on the comment and all that. Sounds good.  Won't touch it for now as Jonald wants to see if maybe we can redo this whole mechanism...